AssignementTransaction/AssignementTransaction/Program.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Data.SqlClient;
+using System.Data.SqlTypes;
+using System.Text;
+
+namespace AssignementTransaction
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Getting connection...");
+
+            SqlConnection connection = new SqlConnection("Data Source=localhost;Initial Catalog=Company;"
+        + "Integrated Security=true;");
+            try
+            {
+                Console.WriteLine("Openning connection...");
+                connection.Open();
+                Console.WriteLine("Connection succesfull!");
+
+                void makeTransaction(string id, string iban, SqlMoney amount)
+                {
+                    StringBuilder sb = new StringBuilder();
+                    sb.Append("exec doTransaction @id_sender = " + id + ", @iban_reciver = " + iban + ", @amount = " + amount);
+
+                    string sqlQuery = sb.ToString();
+                    using (SqlCommand command = new SqlCommand(sqlQuery, connection))
+                    {
+                        using (SqlDataReader reader = command.ExecuteReader())
+                        {
+                            while (reader.Read())
+                            {
+                                Console.WriteLine(reader[0].ToString());
+
+                            }
+                        }
+                    }
+                }
+
+                void ConsoleMenu()
+                {
+                    bool running = true;
+                    while (running)
```

Comment:
Instead of using the running variable, you could've used "while(true)" and then break the loop when "x" is read.

Reply:
I cannot use while(true) with switch. see: https://stackoverflow.com/questions/47251045/while-loop-in-c-sharp-with-switch-statement